### PR TITLE
add cluster-admins for jerry

### PR DESCRIPTION
**Pull request by durandom**
@HumairAK not sure if this is needed, if we get cluster-admin via SSO?!

### CODE REVIEW

File: cluster-scope/overlays/prod/emea/jerry/groups/cluster-admins.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: cluster-admins
+users:
+  - durandom
+  - humairak
```
Comment (HumairAK) [inline cluster-scope/overlays/prod/emea/jerry/groups/cluster-admins.yaml:7]: can you remove lower case (this is a left over from migration, we need to at some point do this for all groups)

**Comment by durandom** [comment]
@HumairAK if I login to https://console-openshift-console.apps.jerry.ionos.emea.operate-first.cloud/ then I'm *not* cluster-admin :( Do you know why?

**Comment by schwesig** [comment]
fixes https://github.com/operate-first/operations/issues/494

**Comment by durandom** [comment]
> @HumairAK if I login to [console-openshift-console.apps.jerry.ionos.emea.operate-first.cloud](https://console-openshift-console.apps.jerry.ionos.emea.operate-first.cloud/) then I'm _not_ cluster-admin :( Do you know why?

https://argocd.operate-first.cloud/applications/argocd-infra is not on auto-sync. Had to manually sync

**Pull request merged**